src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -15,4 +15,35 @@ class Wallet(private val horizonUrl: String, private val networkPassphrase: Stri
     val keypair: KeyPair = KeyPair.random()
     return AccountKeypair(keypair.accountId, String(keypair.secretSeed))
   }
+
+  // Fund (activate) account
+  // TODO: ??? do we need to include add trustline operation here?
+  fun fund(
+    sourceAddress: String,
+    destinationAddress: String,
+    startingBalance: String = "1",
+    sponsorAddress: String = ""
+  ): Transaction {
+    val isSponsored = sponsorAddress.isNotBlank()
+
+    if (!isSponsored && startingBalance.toInt() < 1) {
+      throw Error("Starting balance must be at least 1 XLM for non-sponsored accounts")
```

Comment:
It may be helpful to start defining our own exceptions so that the caller can handle the exceptions according to the type of exception that we throw.

Reply:
:100: I'll add a note to do that once we have more exceptions to consider.